app/controllers/admin/reports_controller.rb

```diff
@@ -27,6 +39,15 @@ def index
         }
       end
 
```

Comment:
I tried that at first, it was really weird tho and giving me a `reports_methods is nil` error :( Will give it another shot, agree that it would be better as one loop